src/repo/learnocaml_process_exercise_repository.ml

```diff
@@ -60,6 +60,14 @@ let dump_outputs = ref None
 
 let dump_reports = ref None
 
+let dump_dot exs =
+  match !Grader_cli.dump_dot with
+    None -> Lwt.return ()
```

Comment:
Why isn't it an error to try to execute `dump_dot` if `Grader_cli.dump_dot` is undefined?